src/App/Platforms/Android/Handlers/LabelHandlerMappings.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Android.OS;
+using Bit.App.Controls;
+using Microsoft.Maui.Handlers;
+
+namespace Bit.App.Handlers
+{
+    public class LabelHandlerMappings
+    {
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Interoperability", "CA1416:Validate platform compatibility", Justification = "<Pending>")]
+        public static void Setup()
+        {
+            Microsoft.Maui.Handlers.LabelHandler.Mapper.AppendToMapping("CustomLabelHandler", (handler, label) =>
+            {
+                if (label is CustomLabel customLabel && customLabel.FontWeight.HasValue && Build.VERSION.SdkInt >= BuildVersionCodes.P)
+                {
+                    handler.PlatformView.Typeface = Android.Graphics.Typeface.Create(null, customLabel.FontWeight.Value, false);
+                    return;
+                }
+
+                if (label is SelectableLabel)
+                {
+                    handler.PlatformView.SetTextIsSelectable(true);
+                }
+            });
+
+            Microsoft.Maui.Handlers.LabelHandler.Mapper.AppendToMapping(nameof(ILabel.AutomationId), (handler, label) =>
+            {
+                handler.PlatformView.ContentDescription = label.AutomationId;
+            });
+
+            // WORKAROUND: There is an issue causing Multiline Labels that also have a LineBreakMode to not display text properly. (it truncates text on first line even with space available)
+            // MAUI Github Issue: https://github.com/dotnet/maui/issues/14125 and https://github.com/dotnet/maui/pull/14918
```

Comment:
This issue has been recently fixed.

Reply:
That is marked as it's going to be released in SR2 and it's not out yet.

Reply:
I thought there was one, but if not yes if you can create it please 🎉